packages/twoslash/src/regexp.ts

```diff
@@ -2,7 +2,7 @@ export const reConfigBoolean = /^\/\/\s?@(\w+)$/mg
 export const reConfigValue = /^\/\/\s?@(\w+):\s?(.+)$/mg
 export const reAnnonateMarkers = /^\s*\/\/\s*\^(\?|\||\^+)( .*)?$/mg
 
-export const reCutBefore = /^\/\/\s?---cut(-before)?---\r?\n/mg
-export const reCutAfter = /^\/\/\s?---cut-after---$/mg
-export const reCutStart = /^\/\/\s?---cut-start---$/mg
-export const reCutEnd = /^\/\/\s?---cut-end---\r?\n/mg
+export const reCutBefore = /^[\t\v\f ]*\/\/\s?---cut(-before)?---\r?\n/mg
```

Comment:
```ts
export const reCutBefore =/^[\t\v\f\uFEFF\p{Zs}]*\/\/\s?---cut(-before)?---\r?\n/mgu
```
Accurately, the regex needs to be like this to include other characters that are treated as white space. https://developer.mozilla.org/en-US/docs/Glossary/Whitespace#in_javascript
But I guess the `u` flag has a perf impact and people won't use those characters.